client/components/applePay/html/README.md

```diff
@@ -0,0 +1,233 @@
+   ```
+
+3. **Start ngrok tunnel:**
+
+   ```bash
+   ngrok http 3000
+   ```
+
+4. **Note your HTTPS URL:**
+   - ngrok will display output similar to:
+
+   ```
+   Forwarding  https://example.ngrok-free.app -> http://localhost:3000
+   ```
+
+   - Copy the domain (e.g. `example.ngrok-free.app`)
+   - This is the domain that needs to be added to the vite server config and registered for Apple Pay in your PayPal developer account
+
+5. **Update Vite configuration:**
+   - Open `applePay/html/vite.config.js`
+   - Add your ngrok domain to the allowed hosts:
+
+   ```javascript
+   export default defineConfig({
+     server: {
+       allowedHosts: ["example.ngrok-free.app"],
+     },
+   });
+   ```
+
+6. **Register ngrok domain with PayPal:**
+   - In your PayPal developer dashboard, navigate to your app's Apple Pay settings under **Features** -> **Accept payments** -> **Apple Pay** -> **Manage**
+   - Add your ngrok domain (e.g. `example.ngrok-free.app`) to the approved domains list
+   - Before clicking **Register Domain**, download and save the domain association file to `applePay/html/.well-known/apple-developer-merchantid-domain-association`
+
+![Screenshot](images/manage-apple-pay.jpg)
+![Screenshot](images/domain-registration-with-paypal.jpg)
+
+**Important Notes:**
+
+- Free ngrok accounts get a new random domain each time you restart ngrok
+- Each new domain needs to be registered for Apple Pay with PayPal
+- The tunnel will stay active as long as the ngrok process is running
+- Always use the HTTPS URL provided by ngrok for Apple Pay testing
+
+### Server Setup
+
+1. **Navigate to the server directory:**
+
+   ```bash
+   cd server/node
+   ```
+
+2. **Install dependencies:**
+
+   ```bash
+   npm install
+   ```
+
+3. **Configure environment variables:**
+   Create a `.env` file in the root directory:
+
+   ```env
+   PAYPAL_SANDBOX_CLIENT_ID=your_paypal_sandbox_client_id
+   PAYPAL_SANDBOX_CLIENT_SECRET=your_paypal_sandbox_client_secret
+   ```
+
+4. **Start the server:**
+   ```bash
+   npm start
+   ```
+   The server will run on `https://localhost:8080`
+
+### Client Setup
+
+1. **Navigate to the Apple Pay demo directory:**
+
+   ```bash
+   cd client/components/applePay/html
+   ```
+
+2. **Install dependencies:**
+
+   ```bash
+   npm install
+   ```
+
+3. **Start the development server:**
+   ```bash
+   npm start
+   ```
+   The demo will be available at `http://localhost:3000`
+
+## 🧪 Testing the Integration
+
+1. **Open Safari Browser**
+   - Navigate to your ngrok https domain
+   - click Visit Site
+
+2. **Initiate Apple Pay**
+   - Click the Apple Pay button
+   - The Apple Pay sheet will appear with your test payment method
+
+3. **Complete Payment**
+   - Select the test credit card you added to your sandbox iCloud wallet
+   - Authenticate with Touch ID, Face ID, or device passcode
+   - The payment should process successfully
+
+4. **Verify Results**
+   - Check the browser console logs for order capture details
```

Comment:
I think the transaction will also show in the paypal develop dashboard too? Could be worth mentioning to check there if that works as expected in sandbox